### PR TITLE
check for empty endpoint array

### PR DESCRIPTION
**Pull request by aaryaattrey**

### CODE REVIEW

File: dozer-api/src/grpc/shared_impl/mod.rs [hunk 1]
```diff
@@ -97,6 +97,10 @@ pub fn on_event<T: Send + 'static>(
         ));
     }
 
+    if endpoints.is_empty() {
+        return Err(Status::invalid_argument("empty-endpoints-array"));
```
Comment (chubei) [inline dozer-api/src/grpc/shared_impl/mod.rs:101]: Don't add `-`s.

**Review by chubei — CHANGES_REQUESTED** [review]

File: dozer-api/src/grpc/shared_impl/mod.rs (hunk 1 above)
Comment (aaryaattrey) [reply dozer-api/src/grpc/shared_impl/mod.rs:101]: Spaces are not handled properly in the output. It is shown like this in output: `empty%20endpoints%20array`. Is that fine?

Comment (chubei) [reply dozer-api/src/grpc/shared_impl/mod.rs:101]: I believe it's frontend problem that the message is not properly displayed. Try postman?

Comment (aaryaattrey) [reply dozer-api/src/grpc/shared_impl/mod.rs:101]: Right. Its a frontend issue.

**Pull request merged**